router/config.go

```diff
@@ -200,9 +202,10 @@ func (c *Config) verify() error {
 	return nil
 }
 
-func (c *Config) LoadFile(filename string) error {
+func (c *Config) LoadFile(filename string, cMode bool) error {
+	containerMode = cMode
```

Comment:
I'm not 100% sure here, but I suspect it will be better to just set the global `containerMode` the moment it becomes known, rather than delegating the assigning of it to some particular function. Don't try and hide the global. Just embrace it.

Reply:
Happy to do that. Just struggled to get access to the global variable from within the router.go file. Even though we are importing the github.com/IMQS/router-core/router package. Maybe there's something I'm missing though.